### PR TITLE
Add --output option to cargo-verify

### PR DESCRIPTION
**Pull request by alastairreid**

### CODE REVIEW

File: cargo-verify/src/main.rs [hunk 1]
```diff
@@ -395,6 +404,14 @@ fn verify(opt: &Opt, package: &str, target: &str) -> CVResult<Status> {
     info_at!(&opt, Verbosity::Informative, "  Building {} for verification", package);
     let bcfile = build(&opt, &package, &target)?;
 
+    info_at!(&opt, Verbosity::Informative, "  Generated binary {}", bcfile.to_string_lossy());
```
Comment (fshaked) [inline cargo-verify/src/main.rs]: In the `structopt` comment for `output` you use "LLVM bitcode file", I think that is better than "binary" here (and a few lines below).

File: cargo-verify/src/main.rs [hunk 2]
```diff
@@ -395,6 +404,14 @@ fn verify(opt: &Opt, package: &str, target: &str) -> CVResult<Status> {
...
+
+    if let Some(output) = &opt.output {
+        std::fs::copy(bcfile, output)?;
+        info_at!(&opt, Verbosity::Informative, "Wrote binary to {}", output.to_string_lossy());
+        return Ok(Status::Verified)
```
Comment (fshaked) [inline cargo-verify/src/main.rs]: Maybe use `exit(0)` instead of `return`? This will avoid printing the "VERIFICATION_RESULT: {}" which is misleading.

**Review by fshaked — CHANGES_REQUESTED** [review]

**Comment by fshaked** [comment]
Another thing, the new `output` option has no effect when using proptest. Either make it an illegal combination (report an error), or implement it for proptest.

File: cargo-verify/src/main.rs (hunk 1 above)
Comment (alastairreid) [reply cargo-verify/src/main.rs]: Good idea - done

File: cargo-verify/src/main.rs (hunk 2 above)
Comment (alastairreid) [reply cargo-verify/src/main.rs]: Good idea - done

**Pull request merged**